### PR TITLE
Closes #1461 - Updates Compare-Object OUTPUT

### LINKED ISSUE #1461: Compare-Object's -PassThru switch is documented incorrectly
Issue Details
-------------

The current docs state the following or slight variations in the description of the `-PassThru` switch:

> Returns an object representing the item with which you are working. By default, this cmdlet does not generate any output.

In reality:

* the cmdlet _does_ generate output (in the general case)

* the true purpose of `-PassThru` is to _omit the custom-object wrapper around the output objects_ (the object that contains the `.SideIndicator` property)

Version(s) of document impacted
------------------------------
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
-  [x] Impacts 3.0 document

### LINKED ISSUE #1464: Help for New-ItemProperty needs additional examples
Issue Details
-----------------
The acceptable values for the PropertyType parameter are not enumerated by intellisense.  Those values are discoverable in the parameters section of the help.  But since the values (at least for the registry provider) are not quite you might expect if you are used to looking at Regedit, I think it would be helpful to have at least one example of creating a new registry value that is of an atypical type, such as "MultiString"

`New-ItemProperty -Path "HKLM:\SOFTWARE\Mine\SomeApp\" -Name 'Server' -PropertyType MultiString -Value ('a','b','c')`

Version(s) of document impacted
------------------------------
- [X] Impacts 6 document
- [X] Impacts 5.1 document
- [X] Impacts 5.0 document
- [X] Impacts 4.0 document
- [X] Impacts 3.0 document

Reason(s) for not selecting all version of documents
----------------------------------------------------
- [ ] The documented feature was introduced in selected version of PowerShell
- [ ] This issue only shows up in selected version of the document

### PR DESCRIPTION
**Pull request by bobbytreed**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [ ] Impacts 6.0 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

File: reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md [hunk 1]
```diff
@@ -223,20 +229,27 @@ Accept wildcard characters: False
...
+
+### System.Management.Automation.PSCustomObject
 
-When you use the PassThru parameter, Compare-Object returns the objects that differed.
-Otherwise, this cmdlet does not generate any output.
+If the objects are different, `Compare-Object` wraps the differing objects in a `PSCustomObject` wrapper with a `SideIndicator` property to reference the differences.
+
+### Other
+
+When you use the *PassThru* parameter, **Compare-Object** omits the `PSCustomObject` wrapper around the compared objects and returns the differing objects, unchanged.
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md]: This should text should also appear as the description of the -PassThru parameter.

Also, the formatting of this is all wrong. Per the style guide:

- When talking about a cmdlet in paragraph, use (`) to highlight cmdlet names
- Property names and parameter names should be in bold

File: reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md [hunk 2]
```diff
@@ -223,20 +229,27 @@ Accept wildcard characters: False
...
+
+If the objects are the same, nothing is returned.
+
+### System.Management.Automation.PSCustomObject
 
-When you use the PassThru parameter, Compare-Object returns the objects that differed.
-Otherwise, this cmdlet does not generate any output.
+If the objects are different, `Compare-Object` wraps the differing objects in a `PSCustomObject` wrapper with a `SideIndicator` property to reference the differences.
+
+### Other
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md]: Get rid of this H3. The next paragraph should be merged with the paragraph above.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1461/reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/compare-object?view=powershell-3.0&branch=pr-en-us-2628) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Compare-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1461/reference/4.0/Microsoft.PowerShell.Utility/Compare-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/compare-object?view=powershell-4.0&branch=pr-en-us-2628) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Compare-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1461/reference/5.0/Microsoft.PowerShell.Utility/Compare-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/compare-object?view=powershell-5.0&branch=pr-en-us-2628) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Utility/Compare-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1461/reference/5.1/Microsoft.PowerShell.Utility/Compare-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/compare-object?view=powershell-5.1&branch=pr-en-us-2628) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Utility/Compare-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1461/reference/6/Microsoft.PowerShell.Utility/Compare-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/compare-object?view=powershell-6&branch=pr-en-us-2628) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C28%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806281622368134-2628%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=1fPZSFv%2BQ7YsyM6qgIJLjhtVTQtFc%2FXOEin8y%2FgxXsw%3D&st=2018-06-28T16%3A28%3A20Z&se=2018-07-29T16%3A33%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md [hunk 3]
```diff
@@ -165,6 +165,8 @@ Accept wildcard characters: False
 Returns an object representing the item with which you are working.
 By default, this cmdlet does not generate any output.
 
+When you use the **PassThru** parameter, `Compare-Object` omits the `PSCustomObject` wrapper around the compared objects and returns the differing objects, unchanged.
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md]: This paragraph is supposed the current paragraph that describes the -PassThru parameter.

File: reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md [hunk 4]
```diff
@@ -245,11 +247,9 @@ If the objects are the same, nothing is returned.
 
 ### System.Management.Automation.PSCustomObject
 
-If the objects are different, `Compare-Object` wraps the differing objects in a `PSCustomObject` wrapper with a `SideIndicator` property to reference the differences.
-
-### Other
+If the objects are different, `Compare-Object` wraps the differing objects in a `PSCustomObject` wrapper with a **SideIndicator** property to reference the differences.
 
-When you use the *PassThru* parameter, **Compare-Object** omits the `PSCustomObject` wrapper around the compared objects and returns the differing objects, unchanged.
+When you use the **PassThru** parameter, `Compare-Object` omits the `PSCustomObject` wrapper around the compared objects and returns the differing objects, unchanged.
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md]: Don't make this a separate paragraph. Join it to the previous paragraph (no blank line between).

Comment (bobbytreed) [reply reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md]: okay, done.

File: reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md (hunk 3 above)
Comment (bobbytreed) [reply reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md]: replaced

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1461/reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/compare-object?view=powershell-3.0&branch=pr-en-us-2628) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Compare-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1461/reference/4.0/Microsoft.PowerShell.Utility/Compare-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/compare-object?view=powershell-4.0&branch=pr-en-us-2628) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Compare-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1461/reference/5.0/Microsoft.PowerShell.Utility/Compare-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/compare-object?view=powershell-5.0&branch=pr-en-us-2628) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Utility/Compare-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1461/reference/5.1/Microsoft.PowerShell.Utility/Compare-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/compare-object?view=powershell-5.1&branch=pr-en-us-2628) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Utility/Compare-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1461/reference/6/Microsoft.PowerShell.Utility/Compare-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/compare-object?view=powershell-6&branch=pr-en-us-2628) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C28%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806281658203832-2628%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=CbUBPVkJBlEmWMJJFftchohuRB1mTFs0%2FYoF1zHjB1k%3D&st=2018-06-28T17%3A03%3A23Z&se=2018-07-29T17%3A08%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**